x-pack/plugins/enterprise_search/public/applications/shared/layout/nav.tsx

```diff
@@ -55,6 +56,25 @@ export const useEnterpriseSearchNav = () => {
         defaultMessage: 'Content',
       }),
     },
+    {
+      id: 'analytics',
+      items: [
+        {
+          id: 'analytics_collections',
+          name: i18n.translate('xpack.enterpriseSearch.nav.analyticsTitle', {
```

Comment:
To fix your build error

```suggestion
      name: i18n.translate('xpack.enterpriseSearch.nav.analyticsCollectionsTitle', {
```